Week-2/Assignment2/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Animation</title>
+</head>
+
+<body>
+    <div>
+        <div class="main">
+            <h1>Hi!</h1>
+            <div class="first">
+                <div class="box red"></div>
+                <div class="box blue"></div>
+            </div>
+            <div class="second">
+                <div class="box yellow"></div>
```

Comment:
class `first` and `second` is not required here. You can use a common class here. Because the properties are same.